packages/dx-grid-core/src/plugins/paging-state/computeds.js

```diff
@@ -1,8 +1,16 @@
-export const paginate = (rows, pageSize, page) =>
-  rows.slice(pageSize * page, pageSize * (page + 1));
+const showAllPagesKey = 'all';
```

Comment:
Shouldn't we export this const and use it other files (where we reference the 'all' value)?